tools/benchmark_tool/openvino/tools/benchmark/utils/utils.py

```diff
@@ -220,34 +255,36 @@ def process_help_inference_string(benchmark_app, device_number_streams):
 
 
 def dump_exec_graph(exe_network, exec_graph_path):
-    try:
-        exec_graph_info = exe_network.get_exec_graph_info()
-        exec_graph_info.serialize(exec_graph_path)
-        logger.info(f'Executable graph is stored to {exec_graph_path}')
-        del exec_graph_info
-    except Exception as e:
-        logger.exception(e)
+    pass # TODO: is there any way to do it in new api?
```

Comment:
What is a problem with having it in new API? We must have this functionality (for WB, for example)

Reply:
There is no problem anymore, i forgot to remove this comment, but serializer api for python still in the work, i will update this function later.